src/quo/components/utilities/token/loader.cljs

```diff
@@ -8,7 +8,7 @@
   [token]
   (let [token-symbol (cond-> token
                        (keyword? token) name
-                       :always          string/lower-case)]
+                       :always          (comp string/lower-case str))]
```

Comment:
string/lower-case crashes with `nil`. discussed with @ulisesmac and he came up with this approach to handle it 👌

Reply:
@J-Son89 I think this needs to be reverted. Currently, no icons are displayed. 

I tried passing `nil` to this function and it works fine. I think we have a condition `(keyword? token)` which will evaluate false for nil, therefore `string/lower-case` wouldn't be called on a nil value.

Reply:
Thanks @OmarBasem. Spotted this yesterday with @ulisesmac and fixed in the send pr

Reply:
It was breaking with nil btw, I wouldn't have added the code otherwise 👌